cmake/utils.cmake

```diff
@@ -209,6 +209,49 @@ function(mayaUsd_get_unittest_target unittest_target unittest_basename)
     set(${unittest_target} "${unittest_name}" PARENT_SCOPE)
 endfunction()
 
+function(mayaUsd_promoteHeaderListWithSubdir)
```

Comment:
Weird, this was added as part of #255

Reply:
If I recall correctly, at the time your branch `kxl-adsk/MAYA-102925/backport_mayareference_schema_refactoring` wasn't merged in and was still in review. I took mayaUsd_promoteHeaderListWithSubdir from your branch for sake of saving time and added it as part of this PR.

Reply:
GitHub tricked me, I thought you already merged dev here since I didn't see any conflicts. All good, thx!